subsys/bluetooth/controller/hci_internal.c

```diff
@@ -642,6 +642,8 @@ static void vs_supported_commands(sdc_hci_vs_supported_vs_commands_t *cmds)
 	cmds->set_power_control_request_params = 1;
 
 #endif
+	cmds->set_conn_event_trigger = 1;
```

Comment:
Add deps to BT_CONN?

Reply:
I'm not sure I understand? This feature currently gets added to all SDC variants and can be used even without connections (for instance with scanning)

Reply:
get_next_conn_event_counter  is for connections only, right?